README.md

```diff
@@ -229,9 +229,9 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 * [Gemini](https://macpaw.com/gemini) - Intelligent duplicate file finder.
 * [Hex Fiend](https://ridiculousfish.com/hexfiend/) - Fast and clever open source hex editor. [![Open-Source Software][OSS Icon]](https://github.com/ridiculousfish/HexFiend/) ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://itunes.apple.com/app/hex-fiend/id1342896380)
 * [Hosts.prefpane](https://github.com/specialunderwear/Hosts.prefpane) - System preference pane to manage your hosts file. [![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/specialunderwear/Hosts.prefpane)
-* [iHosts](https://toolinbox.net/en/iHosts/) - The only `/etc/hosts` editor on Mac App Store. ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://itunes.apple.com/app/id1102004240?mt=12)
+* [iHosts](https://toolinbox.net/iHosts/) - The only `/etc/hosts` editor on Mac App Store. ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://itunes.apple.com/app/id1102004240?mt=12)
```

Comment:
```suggestion
```

Reply:
This looks like malware to me.

Reply:
@alichtman Take a look, it is convenient to develop and modify the host file.

Reply:
I can't read the writing on the website to really understand what it does. I'm not sure why you would need a whole separate application to edit a text file... TextEdit, vim, Sublime, VSCode, etc all exist and will be on a developer's laptop.

Reply:
@alichtman The location of the host file is easy to forget. Editing requires permissions. There is also code highlighting and host configuration backup.

Reply:
@alichtman In fact, I find this software a bit redundant.

Reply:
If you really want to keep it there, you can. I just think it's completely redundant and unnecessary.

Reply:
I don't think it's redundant. In China, a lot of websites are blocked by the Chinese government and one way to bypass the blocking is editing the hosts file to avoid DNS pollution. And not everyone is professional for the editor and file under `/etc`.